core/src/it/scala/org/ensime/fixture/EnsimeConfigFixture.scala

```diff
@@ -29,13 +29,18 @@ trait EnsimeConfigFixture {
   def withEnsimeConfig(testCode: EnsimeConfig => Any): Any
 
   // convenience method
-  def main(lang: String)(implicit config: EnsimeConfig): File =
-    config.projects.head.sources.filter { dir =>
+
+  def mains(lang: String)(implicit config: EnsimeConfig): List[File] =
```

Comment:
No, it hasn't. The `scalaMain` and `javaMain` functions still get one src/main/scala directory. I extracted out this `mains` function as it could be used to get multiple src/main/scala directories for a multi-module testing project.